code/game/machinery/camera/camera.dm

```diff
@@ -65,6 +65,12 @@
 	if(!c_tag)
 		var/area/A = get_area(src)
 		c_tag = "[A ? A.name : "Unknown"] #[rand(111,999)]"
+		if(length(used_ctags) >= 888) // The max number of unique ids from 111 to 999 inclusive
+			error("[get_area(src)] has over the maximum number of tagless cameras, and has run out of IDs!")
+		else	
+			while(c_tag in A.used_ctags))
+				c_tag = "[A ? A.name : "Unknown"] #[rand(111,999)]"
+			A.used_ctags += c_tag

```

Comment:
This is scary since it will take longer and longer for each camera as they init in order, and can, if you're really unlucky, never finish.

Isn't there a better way?

Reply:
Maybe if you collide then just subtract/add to the number repeatedly until you find a free one, and if you wrap and arrive back at the original one, you can break and give up

Reply:
Could also just have them go in sequence, but then they wouldn't be random.

Reply:
I'm ok with that honestly. I don't care thaaaaaaat much. Though that does mean if someone adds ONE new camera to the map, the order gets lost if people have memorized it. But eh.